src/components/navButton/NavButton.tsx

```diff
@@ -0,0 +1,22 @@
+import Button from "@mui/material/Button";
+
+interface Props {
+    page: string;
+    description: string
+}
+
+export default function NaButton({ page, description }: Props) {
```

Comment:
Typo in NavButton here!

Reply:
We should also specify the Props name and call them NavButtonProps. Awesome job abstracting this data and making the component reusable, though.